src/partials-js/data_fetch.js

```diff
@@ -4,8 +4,9 @@ import axios from "axios";
 const $citySearchField = document.querySelector("form input");
 const $citySearchBtn = document.querySelector("form button");
 
+let citiesData = [];
+
```

Comment:
to niepotrzebnie wyciągniete na wierzch

Reply:
Jeżeli zmienna ma być dostępna globalnie i żebyśmy mogli jej używać bez wywoływania funkcji getCityWeather która wysyła kolejne zapytania, to chyba musi. Chyba, że nie to proszę rozwiń to.